internal/sink/sink.go

```diff
@@ -27,84 +30,141 @@ import (
 )
 
 type Sink interface {
-	Update(ancla.Register) error
```

Comment:
still have the update function - but the signature has been updated and is no longer required for the Sink interface